src/stories/Header.tsx

```
@@ -0,0 +1,71 @@
+import React from "react";
+
+import { Button } from "./Button";
```

ESLint

[error] Missing file extension for "./Button"

(import/extensions)